includes/class-wc-stripe-feature-flags.php

```diff
@@ -187,6 +187,10 @@ public static function is_spe_available() {
 	 * @return bool
 	 */
 	public static function is_oc_available() {
-		return 'yes' === self::get_option_with_default( self::OC_FEATURE_FLAG_NAME );
+		return apply_filters(
+			'wc_stripe_is_optimized_checkout_available',
+			'yes' === self::get_option_with_default( self::OC_FEATURE_FLAG_NAME ),
+			self::get_option_with_default( self::OC_FEATURE_FLAG_NAME )
+		);
```

Comment:
Nit: This feels really complicated due to us calling `get_option_with_default` twice. Can we push that logic into a variable so it's "done" before we trigger the filter?
```suggestion
		$feature_flag_value = self::get_option_with_default( self::OC_FEATURE_FLAG_NAME );
		return apply_filters(
			'wc_stripe_is_optimized_checkout_available',
			'yes' === $feature_flag_value,
			$feature_flag_value
		);
```